src/coord/src/command.rs

```diff
@@ -281,7 +283,10 @@ pub enum ExecuteResponse {
     /// The specified prepared statement was created.
     Prepare,
     /// Rows will be delivered via the specified future.
-    SendingRows(#[derivative(Debug = "ignore")] RowsFuture),
+    SendingRows(
+        #[derivative(Debug = "ignore")] RowsFuture,
+        #[derivative(Debug = "ignore")] OtelContext,
+    ),
```

Comment:
These anonymous tuples get confusing fast. Let's use a struct variant instead!

```suggestion
    SendingRows {
       #[derivative(Debug = "ignore")] 
        future: RowsFuture,
        #[derivative(Debug = "ignore")] 
        otel_ctx: OtelContext,
    },
```